.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi**, and look through [known issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=) before submitting.
+
+Our wiki may be found [here](https://github.com/alexa-pi/AlexaPi/wiki). Please spend a bit of time to review this before filing an issue.
```

Comment:
Something like "Check out our [wiki]." - making links correctly rather than [here].
Also link to Audio debugging specifically.